release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -1266,6 +1271,8 @@ With this release, replacement control plane nodes are assigned to the correct i
 
 * Previously, the `oc-mirror` command built the catalog content in the console for OCI and FBC Operators from a mirrored disk image. Consequently, not all content of the catalog was mirrored so some content was missing from the catalog. With this update, the catalog image is built to reflect the mirrored content prior to pushing it to the destination registry resulting in a more complete catalog. (link:https://issues.redhat.com/browse/OCPBUGS-5805[*OCPBUGS-5805*])
 
+* Previously, the `oc-mirror` command added the operator catalog as an entry in the `ImageContentSourcePolicy`. With this update, the operator catalog is consumed directly from the destination registry because of the `CatalogSource` making the additon of the operator catalog in the `ImageContentSourcePolicy` is not needed. (link:https://issues.redhat.com/browse/OCPBUGS-10320[*OCPBUGS-10320*])
```

Comment:
The last part of the second sentence is a little confusing imo. I think you can just remove it
```suggestion
* Previously, the `oc-mirror` command added the operator catalog as an entry in the `ImageContentSourcePolicy`. With this update, the operator catalog is consumed directly from the destination registry.(link:https://issues.redhat.com/browse/OCPBUGS-10320[*OCPBUGS-10320*])
```